docs/proposals/002-routing-processor-proposal/routing-processor.proto

```diff
@@ -0,0 +1,188 @@
+syntax = "proto3";
+
+package routing_processor.v1alpha1;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
```

Comment:
let's not use the third party protobuf dependency and start with a really simple self contained definition.

Reply:
dependency removed now